### ISSUE TITLE
[SUGGESTION] | Divide northern Kalimdor into kingdoms

### ISSUE THREAD

**Issue by TheGuyNamedMoney** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#237]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**
I am not sure if it has been suggested before. But i'll do so anyway. Northern Kalimdor is currently only 3 kingdoms, Ashenvale, Hyjal and Kaldar. (Picture related)
https://cdn.discordapp.com/attachments/509863847818035203/555381892971626496/unknown.png

My suggestion is to split it down further into Kingdoms. Like Splitting Kaldar into Azuremyst, Bloodmyst and Teldrassil.
Splitting Ashenvale into Ashenvale, Duskwood, Felwood and Mooglade.
And finally splitting Hyjal into Hyjal and Winterspring.

The issue with such massive kingdoms is that they become exessively difficult to control. As you don't want to split that much land up into Dutchies, but at the same time don't want to give one King in your empire -that- much power, or they are sure to cause trouble and rebel.

Thank you for taking the time to listen to one nerd's suggestion, loving the mod so far :)

**Comment by zumbak04** [comment]
De Jure map I got:
![image](https://user-images.githubusercontent.com/46384992/54677523-7c6a0080-4b1c-11e9-9a02-d47b0e6baf31.png)

> My suggestion is to split it down further into Kingdoms. Like Splitting Kaldar into Azuremyst, Bloodmyst and Teldrassil.

These kingdoms would be too small and have 1 duchy. Anyway, I've transferred Kalidar Duchy to Darkshore Kingdom. There's two reasons: first, Kalidar AKA Teldrassil Isle is mentioned as a part of Darkshore in the lore, second, those who will live on Azuremyst Isles won't have de jure claims on the elves.
I won't close this issue for a while to hear your suggestions.
